sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/eventhubs/AzureEventHubsMessagingAutoConfiguration.java

```diff
@@ -121,16 +122,29 @@ public EventHubsProducerFactory defaultEventHubsNamespaceProducerFactory(
         }
 
         /**
-         * Creates an Event Hubs message converter.
+         * Creates an Event Hubs message converter with an isolated ObjectMapper.
          *
          * @return An Event Hubs message converter.
          */
         @Bean
         @ConditionalOnMissingBean
-        public EventHubsMessageConverter eventHubsMessageConverter() {
+        @ConditionalOnProperty(value = "spring.cloud.azure.message-converter.isolated-object-mapper.enabled", havingValue = "true", matchIfMissing = true)
+        EventHubsMessageConverter eventHubsMessageConverter() {
             return new EventHubsMessageConverter();
```

Comment:
Why is here different from the design doc? Where's the calling ctor with ObjectMapper instance?

Reply:
In design doc is to make it easier to read, but actually non-ctor use the OM from OMHolder, same as the design.
<img width="331" alt="image" src="https://user-images.githubusercontent.com/92105726/208053428-d2b1b2df-fc63-43a1-9898-75318b50e5d9.png">
<img width="609" alt="image" src="https://user-images.githubusercontent.com/92105726/208053470-f1519ddd-a953-49d9-bd22-b87d880d444f.png">